src/Selenium2Driver.php

```diff
@@ -163,23 +179,13 @@ public function getDesiredCapabilities()
     /**
      * Sets the WebDriver instance
      *
-     * @param WebDriver $webDriver An instance of the WebDriver class
+     * @param RemoteWebDriver $webDriver An instance of the WebDriver class
      */
-    public function setWebDriver(WebDriver $webDriver)
+    public function setWebDriver(RemoteWebDriver $webDriver)
     {
         $this->webDriver = $webDriver;
     }
 
-    /**
-     * Gets the WebDriverSession instance
-     *
-     * @return \WebDriver\Session
-     */
-    public function getWebDriverSession()
```

Comment:
to allow bypassing the Mink abstraction for things which are not supported (accepting prompts for instance), we should expose `$this->webDriver` (which is the equivalent of what `\WebDriver\Session` is in the old library)

Reply:
returned as `getWebDriver`